lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -185,13 +185,13 @@ describe('workers/repository/update/branch/index', () => {
       });
     });
 
-    it('skips branch if not stabilityDays not met', async () => {
+    it('skips branch if not minimumReleaseAge not met', async () => {
```

Comment:
I don't understand this test description. This description has a double negative: `if not` + `not met`. I'll let one of the developers rewrite this description. 😉

Reply:
extra not 🙃

Reply:
Drop the first not

Reply:
@rarkins can you please comment, I don't have the rights to do that. 😉 This is fixed by commit [f2c459c](https://github.com/renovatebot/renovate/pull/21376/commits/f2c459c5606a6c00b8fe75c96d669012531597e9).